app/queries/servers/thread.ts

```diff
@@ -123,3 +125,42 @@ export const queryUnreadsAndMentionsInTeam = (database: Database, teamId: string
         }),
     );
 };
+
+export async function queryNewestThreadInTeam(
```

Comment:
Also the naming part, all the fetches should start with `get`. So this becomes `getNewestThreadInTeam`